projects/vdk-core/tests/vdk/internal/builtin_plugins/termination_message/test_writer.py

```diff
@@ -16,6 +18,95 @@
 from vdk.internal.core.statestore import StateStore
 
 
+class TestWriterIdempotence(unittest.TestCase):
```

Comment:
As far as I can se we are testing here "TerminationMessageWriterPlugin" . Let's name it then "TestTerminationMessageWriterPlugin" ?